.github/workflows/push-event.yml

```diff
@@ -131,4 +132,9 @@ jobs:
           git commit -am "[Auto] Update APK/AAB's from $branch ($(date +%Y-%m-%d.%H:%M:%S))"
           git branch -D apk
           git branch -m apk
-          git push --force origin apk
\ No newline at end of file
+          git push --force origin apk
+
+      - name: Run Release Drafter
+        uses: release-drafter/release-drafter@v6
+        with:
+          version: ${{ steps.android-version.outputs.NEW_VERSION_NAME }}
```

Comment:
Same here. We don't need the new version name. We use the one that's used to build the app